applications/vessel-history/public/locales/fr/flags.json

```diff
@@ -0,0 +1,252 @@
+{
```

Comment:
do we need translations in all the languages for this file?

Reply:
yes, we do